packages/api-v2/types.ts

```diff
@@ -2144,11 +2146,21 @@ type TPrivateSocketEndpoints = {
     };
 };
 
+type TAccountList = NonNullable<AccountListResponse['account_list']>[number] & { excluded_until: Date };
+
+interface extendedAccountListResponse extends AccountListResponse {
```

Comment:
```suggestion
interface ExtendedAccountListResponse extends AccountListResponse {
```

Reply:
maybe we can start naming with capital `E` or name it as `IExtendedAccountListResponse` ?

Reply:
Done.